### PR TITLE
Added stubs around File Repository with Content View

### LINKED ISSUE #4999: Stubs for Content View with Arbitrary File Repository

### PR DESCRIPTION
**Pull request by renzon**
close #4999

### CODE REVIEW

File: tests/foreman/ui/test_contentview.py [hunk 1]
```diff
@@ -5239,3 +5239,102 @@ def test_positive_remove_cv_version_from_multi_env_capsule_scenario(self):
...
+        :caseautomation: notautomated
+
+        :CaseLevel: System
+        """
+
+    @stubbed()
+    @tier2
+    def test_positive_arbitrary_file_repo_promotion(self):
+        """Check a File Repository with Arbitrary File can be removed from a
+        Content View
```
Comment (sghai) [inline tests/foreman/ui/test_contentview.py]: Didn't get this stub. I see two cases here:

Are we trying to promote a CV containing repo w/ arbitrary file ? 
or 
Are we trying to remove repo containing arbitrary file from promoted CV ?

**Comment by sghai** [comment]
ACK, pending a comment.

File: tests/foreman/ui/test_contentview.py (hunk 1 above)
Comment (renzon) [inline tests/foreman/ui/test_contentview.py]: Copying and pasting is pure evil =P hahaha

**Comment by san7ket** [comment]
@sghai I guess your comment is resolved , please re-visit.

**Comment by sghai** [comment]
Thanks @renzon

**Pull request merged**